docs/design/jobflow/README.md

```diff
@@ -125,6 +125,9 @@ The specification of cloud-native services defines service metadata, version lis
 | ----------------- | ------------------------------------ | -------- | ------------- | ------------------------------------------------------------ |
 | `flows`           | [`Flow array`](#Flow) | Y        |               | Describes the dependencies between vcjobs. |
 | `jobRetainPolicy` | `string`                             | Y        | retain | After JobFlow succeed, keep the generated job. Otherwise, delete it. |
+| `jobRetry` | `int`                          | N        | 1 | JobFlow global retry count.|
```

Comment:
This name is not good.
For consistency. Let's use maxRetry.

Reply:
Thanks @hwdef 
Change to global `maxRetry`